Makefile

```diff
@@ -528,6 +549,8 @@ endif
 
 CFLAGS += -D_GNU_SOURCE -ggdb3 -Wall -Wfatal-errors -Werror
 
+CPPFLAGS += --std=c++11
```

Comment:
See - here is a soft-benefit, if you will, that this entire tooling can work on even on an older rev-level of C++.

This is unlike the `source_location{}` class stuff that needs C++20.